src/gauge/radial-gauge/radial-gauge.component.ts

```diff
@@ -0,0 +1,266 @@
+import { Component, AfterViewInit, ViewEncapsulation, 
+  ChangeDetectionStrategy, Input, Output, EventEmitter } from '@angular/core';
+
+import { ScaleLinear, scaleLinear } from 'd3-scale';
+import { line, curveLinear, Line } from 'd3-shape';
+
+import { BaseChartComponent } from './../../common/base-chart.component';
+import { ColorHelper } from '../../common/color.helper';
+
+@Component({
+  selector: 'ngx-charts-radial-gauge',
+  templateUrl: './radial-gauge.component.html',
+  styleUrls: ['./radial-gauge.component.css'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class RadialGaugeComponent extends BaseChartComponent implements AfterViewInit {
+
+  public arcs = [];
+  public dimensions: any = {};
+  public displayingValue: string;
+  
+  @Input() public displayValue: string;
+  @Input() public showValue: boolean = true;
+  @Input() public value: number = 20;
+  @Input() public minValue = 0;
+  @Input() public maxValue = 100;
+
+  @Input() public minAngle = -90;
+  @Input() public maxAngle = 90;
+  @Input() public innerArcRadius: number;
+  @Input() public outerArcRadius: number;
+
+  @Input() public majorTicks = 5;
+  @Input() public axisRadius: number;
+
+  @Input() public pointerWidth: number;
+  @Input() public pointerHeadLength: number;
+  @Input() public pointerTailLength: number;
+  @Input() public pointerColor: string;
+  @Input() public segments = [];
+
+  @Output() public click: EventEmitter<any> = new EventEmitter();
+  @Output() public mouseEnter: EventEmitter<any> = new EventEmitter();
+  @Output() public mouseLeave: EventEmitter<any> = new EventEmitter();
+
+  @Output() public arcClick: EventEmitter<any> = new EventEmitter();
+  @Output() public arcMouseEnter: EventEmitter<any> = new EventEmitter();
+  @Output() public arcMouseLeave: EventEmitter<any> = new EventEmitter();
+
+  @Output() public labelClick: EventEmitter<any> = new EventEmitter();
+  @Output() public labelMouseEnter: EventEmitter<any> = new EventEmitter();
+  @Output() public labelMouseLeave: EventEmitter<any> = new EventEmitter();
+
+  @Output() public pointerClick: EventEmitter<any> = new EventEmitter();
+  @Output() public pointerMouseEnter: EventEmitter<any> = new EventEmitter();
+  @Output() public pointerMouseLeave: EventEmitter<any> = new EventEmitter();
+
+  private colors: ColorHelper;
+  private ticks: number[];
+  private degreeRange: number;
+
+  ngAfterViewInit(): void {
+    super.ngAfterViewInit();
+  }
+
+  public update(): void {
+    super.update();
+
+    this.ticks = this.getTicks();
+    this.degreeRange = this.getDegreeRange();
+
+    this.dimensions = this.getDimensions();
+
+    this.displayingValue = !this.checkStringEmpty(this.displayValue) 
+    ? this.displayValue 
+    : this.value.toString();
```

Comment:
The files look to be consistently using: [Object.prototype.toLocaleString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/toLocaleString) instead of [Object.prototype.toString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/prototype) on both `string` and `date` instances.

So I think it would be used like so:
```js
this.displayingValue = !this.checkStringEmpty(this.displayValue) 
? this.displayValue 
: this.value.toLocaleString();
```

As for the `this.checkStringEmpty(this.displayValue)`, ngx-charts looks to be ok with using double bit wise Javascript idioms, so you can remove that function with this one-liner:

```js
this.displayingValue = !!this.displayValue || this.value.toLocaleString()
``` 

But, really you can use that one-liner directly in your `radial-guage.component.html` and remove this entire line, the extra `checkStringEmpty` function and the variable `this.displayingValue` :-)

Within line 49 of radial-guage.component.html
```html
<tspan *ngIf="showValue" x="0" dy="1.5em">{{ !!displayValue || value.toLocaleString() }}</tspan>
```